src/main/java/org/magicdgs/http/jsr203/HttpsFileSystemProvider.java

```diff
@@ -0,0 +1,22 @@
+package org.magicdgs.http.jsr203;
+
+/**
+ * Read-only {@link java.nio.file.spi.FileSystemProvider} for HTTPS.
+ *
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ */
+public final class HttpsFileSystemProvider extends HttpAbstractFileSystemProvider {
```

Comment:
Should be added to `resources/META-INF/services/java.nio.file.spi.FileSystemProvider`

Reply:
Done.